store/sql_store_test.go

```diff
@@ -0,0 +1,110 @@
+package store
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/go-gorp/gorp"
+	"github.com/go-sql-driver/mysql"
+	"github.com/mattermost/mattermost-mattermod/model"
+)
+
+const (
+	defaultMysqlDSN     = "mattermod:mattermod@tcp(localhost:3306)/mattermost_mattermod_test?charset=utf8mb4,utf8\u0026readTimeout=30s\u0026writeTimeout=30s\u0026parseTime=true"
+	defaultMysqlUser    = "mattermod"
+	defaultMysqlRootPWD = "mattermod"
+	defaultMysqlUserPWD = "mattermod"
+	defaultMysqlDB      = "mattermod_test"
+)
+
+type teardownFunc func()
+
+func getTestSQLStore(t *testing.T) (*SQLStore, teardownFunc) {
+	teardownSuite, err := createTempDB(defaultMysqlDB, getEnv("MYSQL_USER", defaultMysqlUser))
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Objections on using testify?

Reply:
I think it's not needed here, we don't check for specific errors or specific outputs. We just don't want any errors on the path. Also, I may missing something.

Reply:
`testify` has become the standard way of asserting any output from a test. I personally quite like the way the output looks like and therefore prefer to use it everywere.

Reply:
I see, thanks for the explanation. Let's continue with `testing` and once we complete the tests in the `store` package. We can spike and convert these to use `testify` instead.